iocage_lib/iocage.py

```diff
@@ -1386,9 +1388,11 @@ def get(
 
             return jail_list
 
-    def import_(self):
+    def import_(self, compression_algo='zip'):
```

Comment:
Is there a way to autodetect the file instead?

Reply:
Well we can but that means an exception if we have 2 exported jails with similar names but different algorithms. I guess i raise an exception in that case and that will allow this to automatically proceed if there are no conflicts. Thoughts ?